guides/common/modules/proc_importing-a-subscription-manifest-into-foreman-server.adoc

```diff
@@ -19,10 +24,9 @@ endif::[]
 
 .Procedure
 . In the {ProjectWebUI}, ensure the context is set to the organization you want to use.
-. In the {ProjectWebUI}, navigate to *Content* > *Subscriptions* and click *Manage Manifest*.
-. In the Manage Manifest window, click *Browse*.
+. In the {ProjectWebUI}, navigate to *Content* > *Subscriptions* and click *Import a Manifest*.
```

Comment:
The "Import a Manifest" button only appears when no manifest is imported. It may be better to stick to the old instructions here.

Reply:
Addressed :+1: